desktop/mac/install.md

```diff
@@ -88,6 +90,18 @@ Your Mac must meet the following requirements to install Docker Desktop successf
 
     For more information, see [Docker Desktop License Agreement](../../subscription/index.md#docker-desktop-license-agreement). We recommend that you also read the [Blog](https://www.docker.com/blog/updating-product-subscriptions/){: target="_blank" rel="noopener" class="_" id="dkr_docs_desktop_install_btl"} and [FAQs](https://www.docker.com/pricing/faq){: target="_blank" rel="noopener" class="_" id="dkr_docs_desktop_install_btl"} to learn how companies using Docker Desktop may be affected.
 
+### Install from the command line
+
+After downloading `Docker.dmg`, to install Docker Desktop to the Applications folder, run for instance the following commands in a terminal:
+```
+sudo hdiutil attach Docker.dmg
+sudo /Volumes/Docker/Docker.app/Contents/MacOS/install --accept-license
```

Comment:
I'm not sure we want to put the --accept-license in the example here. I understand why it's there (to show where the arguments go) but I don't want people copy- and pasting it and then saying that they never knowingly accepted the license.

Reply:
Good point, I’ll remove the arguments from the example.